src/python/pants/backend/python/util_rules/local_dists.py

```diff
@@ -98,7 +98,9 @@ async def isolate_local_dist_wheels(
     )
     provided_files = set(wheels_listing_result.stdout.decode().splitlines())
 
-    return LocalDistWheels(tuple(wheels), wheels_snapshot.digest, frozenset(provided_files))
+    return LocalDistWheels(
+        tuple(sorted(wheels)), wheels_snapshot.digest, frozenset(sorted(provided_files))
```

Comment:
Minor (and no ned to change), but... I think a `frozenset` is already order-independent, so there's no need to sort the input.

This suggests another potential solution: use a `frozenset` (or `FrozenOrderedSet`) instead of `tuple` if order truly doesn't matter for all uses. This may be more resilient for future refactorings, because caller can never forget to sort. What do you think?

Reply:
Thanks, I failed to observe that. It could certainly be valuable to go over and see where order is of no consequence and save a few cycles by not sorting.

Saving that as an exercise for the reader/or future me. ;)

Reply:
Lets keep an eye on performance, in case it seems to have degraded for largish repos, it could be from the additional sorting going into this.. which suggests using a version of frozen set may be the better route.